runner/Tests/maya/render-vray-windows/job.parameters2020.json

```diff
@@ -1,12 +1,12 @@
 {
    "jobName":{
-      "value":"maya-vray-windows-2017"
+      "value":"maya-vray-windows-2020"
```

Comment:
add corresponding test to testConfiguration.json

Reply:
yep that one is called "maya2020-vray-windows"